examples/terraform/vsphere/variables.tf

```diff
@@ -263,3 +263,9 @@ variable "ip_family" {
     error_message = "Unsupported IP Family specified."
   }
 }
+
+variable "enable_disk_uuid" {
+  default = true
+  type = bool
+  description = "Expose the UUIDs of attached virtual disks to the virtual machine, allowing access to them in the guest"
+}
```

Comment:
What's the point in having this as an option? Shouldn't we lock this to `true`? IIRC this is a requirement for CSI (and maybe CCM) to work at all.